tests/installation/disable_grub_timeout.pm

```diff
@@ -31,8 +31,9 @@ sub run {
     }
 
     # Workaround for bsc#1070233: not update "Booting" option in upgrade mode
-    return record_soft_failure('bsc#1070233: Error if click on Booting option')
-      if (get_var('UPGRADE') && (!(is_sle && !sle_version_at_least('15')) || !(is_leap && !leap_version_at_least('15.0'))));
+    if (get_var('UPGRADE') && (!is_sle('<15') || !(is_leap && !leap_version_at_least('15.0')))) {
```

Comment:
Now it should be possible to write `!is_sle('<15')` as `is_sle('15+')`, right?

Reply:
`!is_sle('<15')` - is true for also for opensuse, so your 2 statements are not equal